build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'java'
+    id 'idea'
+    id 'org.springframework.boot' version '2.1.7.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+}
+
+group 'org.neorg'
```

Comment:
Is it an appropriate group for the project?

Reply:
It's a very good question. Maybe `com.rt` )

Reply:
I guess you should use the name of your project or maybe a potential web address of the system.
Something like "test.system", "com.test.system", "com.ts", "com.tsys" or "org.*".